packages/sanity/src/core/hooks/useClient.ts

```diff
@@ -2,7 +2,24 @@ import type {SanityClient} from '@sanity/client'
 import type {SourceClientOptions} from '../config'
 import {useSource} from '../studio'
 
-/** @beta */
+/**
+ * React hook that returns a configured Sanity client instance based on the given configuration.
+ * Automatically uses the correct project and dataset based on the current active workspace.
+ *
+ * @public
+ * @param clientOptions - Options for the client. Specifying
+ *   {@link https://www.sanity.io/docs/api-versioning | apiVersion} is required in order to
+ *   prevent breaking changes if studio changes the API version used in other places.
+ * @returns A configured Sanity client instance
+ * @remarks The client instance is automatically memoized based on API version
+ * @example Instantiating a client
```

Comment:
Do you mean "initiating" here?

Reply:
Instantiating as in "Creating an instance of". Maybe a bit technical, but "initiating" sounds wrong too. How about "Creating a client" or "Getting a client"?

Reply:
Aha. Maybe "setting up" or "actualizing"? If not we could just stick with what you wrote first!